libgalois/include/galois/Constants.h

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
```

Comment:
Is it possible to put this code somewhere more specific to the Python bindings?

I don't mind using adding Python bindings as an opportunity to clean up the C++ API but adding `UpdateRequestIndexer` seems too application-specific add to the general C++ library.

Reply:
I agree that UpdateRequestIndexer is too application-specific but this is the best I could do to make Cython generate correct code for OBIM. I will be working with Arthur to find better way to do this.

Reply:
I'd personally lean towards keeping this on a branch (even in this repo) until we can resolve this. We are probably not that far away from doing this in a better way, any collaboration only involves you and Arthur, and `Constants.h` doesn't seem like a good (even if temporary place) to put this code in `libgalois`.

Reply:
I created libpygalois for cython/python related stuff. Is that fine? If not then we can move this to a separate branch.

Reply:
`libpygalois` seems fine to me; I know that @arthurp was looking into some initialization issues that might require rearranging the Cython build, what ever he comes up with might also be a good place to put shared code between Python modules. But you can probably figure this out later.